adam_core/coordinates/tests/test_covariances.py

```diff
@@ -103,3 +103,14 @@ def test_CoordinateCovariances_to_dataframe():
     np.testing.assert_equal(df["sigma_vx"].values, np.sqrt(np.array([25.0, 36.0])))
     np.testing.assert_equal(df["sigma_vy"].values, np.sqrt(np.array([36.0, 49.0])))
     np.testing.assert_equal(df["sigma_vz"].values, np.sqrt(np.array([49.0, 64.0])))
+
+
+def test_CoordinateCovariances_is_all_nan():
```

Comment:
Might want to add the following test scenarios:
- one of the 6x6 cells is nan, but not all of them
- the list is itself nan, rather than all of its elements being nan (CoordinateCovariances.from_arrays([None, np.ones((6, 6)).flatten()]), i think, but I am not sure)

Reply:
Done. And it led to finding a missing corner case in `to_matrix()`. See: [97fb521](https://github.com/B612-Asteroid-Institute/adam_core/pull/30/commits/97fb5218e4efa3f0568f3a4fedf0fd82dbce868f).